### PR TITLE
argyllcms: 2.2.0 -> 2.2.1

### LINKED ISSUE #138924: Update ArgyllCMS
###### Checklist

- [X] Checked the [nixpkgs master branch](https://github.com/NixOS/nixpkgs)

- [X] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

###### Project name
`nix search` name: argyllcms

current version: 2.2.0
desired version: 2.2.1

###### Notify maintainers
last edited by @Artturin

###### Note for maintainers

testing it is only possible with gnome color manager setting, since displaycal is broken

you also need special, expensive hardware

### PR DESCRIPTION
**Pull request by Artturin**
###### Motivation for this change
Closes #138924

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 138953` at bcd383fd run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>displaycal</li>
  </ul>
</details>
<details>
  <summary>4 packages skipped due to time constraints:</summary>
  <ul>
    <li>adapta-gtk-theme</li>
    <li>darktable</li>
    <li>libsForQt512.discover</li>
    <li>libsForQt514.discover</li>
  </ul>
</details>
<details>
  <summary>61 packages built successfully:</summary>
  <ul>
    <li>argyllcms</li>
    <li>chrome-gnome-shell</li>
    <li>cinnamon.cinnamon-common</li>
    <li>cinnamon.cinnamon-control-center</li>
    <li>cinnamon.cinnamon-screensaver</li>
    <li>cinnamon.cinnamon-session</li>
    <li>cinnamon.cinnamon-settings-daemon</li>
    <li>colord</li>
    <li>colord-gtk</li>
    <li>eolie</li>
    <li>fwupd</li>
    <li>gnome-firmware-updater</li>
    <li>gnome.gnome-color-manager</li>
    <li>gnome.gnome-control-center (gnome.gnome_control_center)</li>
    <li>gnome.gnome-disk-utility</li>
    <li>gnome.gnome-session (gnome.gnome_session)</li>
    <li>gnome.gnome-settings-daemon (gnome.gnome_settings_daemon)</li>
    <li>gnome.gnome-settings-daemon338 (pantheon.gnome-settings-daemon)</li>
    <li>gnome.gnome-shell (gnome.gnome_shell)</li>
    <li>gnome.gnome-terminal (gnome.gnome_terminal)</li>
    <li>gnome.gnome-tweaks (gnome.gnome-tweak-tool)</li>
    <li>gnome.gpaste</li>
    <li>gnome.mutter</li>
    <li>gnome.mutter338 (pantheon.mutter)</li>
    <li>gnome.pomodoro</li>
    <li>simple-scan (gnome.simple-scan)</li>
    <li>gnomeExtensions.easyScreenCast</li>
    <li>gnomeExtensions.gsconnect</li>
    <li>libsForQt5.discover (libsForQt515.discover ,plasma5Packages.discover)</li>
    <li>monitor</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-session-settings</li>
    <li>pantheon.gala</li>
    <li>pantheon.switchboard-plug-a11y</li>
    <li>pantheon.switchboard-plug-about</li>
    <li>pantheon.switchboard-plug-mouse-touchpad</li>
    <li>pantheon.switchboard-plug-pantheon-shell</li>
    <li>pantheon.switchboard-plug-power</li>
    <li>pantheon.switchboard-plug-security-privacy</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-indicator-a11y</li>
    <li>pantheon.wingpanel-indicator-bluetooth</li>
    <li>pantheon.wingpanel-indicator-datetime</li>
    <li>pantheon.wingpanel-indicator-keyboard</li>
    <li>pantheon.wingpanel-indicator-network</li>
    <li>pantheon.wingpanel-indicator-nightlight</li>
    <li>pantheon.wingpanel-indicator-notifications</li>
    <li>pantheon.wingpanel-indicator-power</li>
    <li>pantheon.wingpanel-indicator-session</li>
    <li>pantheon.wingpanel-indicator-sound</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>phoc</li>
    <li>phosh</li>
    <li>vimix-gtk-themes</li>
    <li>weston</li>
    <li>westonLite</li>
    <li>whitesur-gtk-theme</li>
    <li>xdg-desktop-portal-gtk</li>
    <li>xiccd</li>
  </ul>
</details>

**Comment by davidak** [comment]
Result of `nixpkgs-review pr 138953` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages marked as broken and skipped:</summary>
  <ul>
    <li>displaycal</li>
    <li>libsForQt512.discover</li>
    <li>libsForQt514.discover</li>
  </ul>
</details>
<details>
  <summary>69 packages built:</summary>
  <ul>
    <li>adapta-gtk-theme</li>
    <li>argyllcms</li>
    <li>chrome-gnome-shell</li>
    <li>cinnamon.cinnamon-common</li>
    <li>cinnamon.cinnamon-control-center</li>
    <li>cinnamon.cinnamon-gsettings-overrides</li>
    <li>cinnamon.cinnamon-screensaver</li>
    <li>cinnamon.cinnamon-session</li>
    <li>cinnamon.cinnamon-settings-daemon</li>
    <li>colord</li>
    <li>colord-gtk</li>
    <li>darktable</li>
    <li>diffoscope</li>
    <li>eolie</li>
    <li>fwupd</li>
    <li>gnome-firmware-updater</li>
    <li>gnome.gnome-color-manager</li>
    <li>gnome.gnome-control-center (gnome.gnome_control_center)</li>
    <li>gnome.gnome-disk-utility</li>
    <li>gnome.gnome-session (gnome.gnome_session)</li>
    <li>gnome.gnome-settings-daemon (gnome.gnome_settings_daemon)</li>
    <li>gnome.gnome-settings-daemon338 (pantheon.gnome-settings-daemon)</li>
    <li>gnome.gnome-shell (gnome.gnome_shell)</li>
    <li>gnome.gnome-software</li>
    <li>gnome.gnome-terminal (gnome.gnome_terminal)</li>
    <li>gnome.gnome-tweaks (gnome.gnome-tweak-tool)</li>
    <li>gnome.gpaste</li>
    <li>gnome.mutter</li>
    <li>gnome.mutter338 (pantheon.mutter)</li>
    <li>gnome.pomodoro</li>
    <li>simple-scan (gnome.simple-scan)</li>
    <li>gnomeExtensions.easyScreenCast</li>
    <li>gnomeExtensions.gsconnect</li>
    <li>gnomeExtensions.night-theme-switcher</li>
    <li>gnomeExtensions.system-monitor</li>
    <li>libsForQt5.discover (libsForQt515.discover ,plasma5Packages.discover)</li>
    <li>monitor</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-gsettings-schemas</li>
    <li>pantheon.elementary-session-settings</li>
    <li>pantheon.gala</li>
    <li>pantheon.switchboard-plug-a11y</li>
    <li>pantheon.switchboard-plug-about</li>
    <li>pantheon.switchboard-plug-mouse-touchpad</li>
    <li>pantheon.switchboard-plug-pantheon-shell</li>
    <li>pantheon.switchboard-plug-power</li>
    <li>pantheon.switchboard-plug-security-privacy</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-indicator-a11y</li>
    <li>pantheon.wingpanel-indicator-bluetooth</li>
    <li>pantheon.wingpanel-indicator-datetime</li>
    <li>pantheon.wingpanel-indicator-keyboard</li>
    <li>pantheon.wingpanel-indicator-network</li>
    <li>pantheon.wingpanel-indicator-nightlight</li>
    <li>pantheon.wingpanel-indicator-notifications</li>
    <li>pantheon.wingpanel-indicator-power</li>
    <li>pantheon.wingpanel-indicator-session</li>
    <li>pantheon.wingpanel-indicator-sound</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>phoc</li>
    <li>phosh</li>
    <li>vimix-gtk-themes</li>
    <li>weston</li>
    <li>westonLite</li>
    <li>whitesur-gtk-theme</li>
    <li>xdg-desktop-portal-gtk</li>
    <li>xiccd</li>
  </ul>
</details>

**Review by davidak — APPROVED** [review]
Change looks good

Builds

tested one binary (dispwin). seem to run

can't test because i don't own a colorimeter

File: pkgs/tools/graphics/argyllcms/default.nix [hunk 1]
```diff
@@ -4,24 +4,22 @@
...
 
   src = fetchzip {
     # Kind of flacky URL, it was reaturning 406 and inconsistent binaries for a
     # while on me. It might be good to find a mirror
     url = "https://www.argyllcms.com/Argyll_V${version}_src.zip";
-    sha256 = "sha256-EcVwYJfJbWWXl58O3ulsrWgUYTgR4uWdMgb0Z140Pu4=";
+    sha256 = "sha256-umY3wQfG26Okqnw+MCUnlwWTAyJ6MR/FHe5oe61KBh0=";
   };
 
-  patches = [ ./gcc5.patch ];
```
[the remark this answers is not in the document]
Comment (Artturin) [reply pkgs/tools/graphics/argyllcms/default.nix:16]: #138996

**Pull request merged**